src/Distribution/Audit.hs

```diff
@@ -46,7 +46,7 @@ import Security.Advisories (Advisory (..), Keyword (..), ParseAdvisoryError (..)
 import Security.Advisories.Cabal (ElaboratedPackageInfoAdvised, ElaboratedPackageInfoWith (..), matchAdvisoriesForPlan)
 import Security.Advisories.Convert.OSV qualified as OSV
 import Security.Advisories.Filesystem (listAdvisories)
-import System.Exit (exitFailure)
+import System.Exit
```

Comment:
I would prefer if we could keep full qualified imports as often as possible, if it isn't like 20 imports, I think it's fine

Reply:
Good point!